integration/app/test-demo/server.js

```diff
@@ -0,0 +1,152 @@
+        response.statusCode = 200;
+        response.setHeader('Content-Type', 'text/html');
+        response.end(fs.readFileSync(`dist/${app}.html`));
+      } else if (
+        request.method === 'POST' &&
+        request.url.startsWith('/join?')
+      ) {
+        const query = url.parse(request.url, true).query;
+        const title = query.title;
+        const name = query.name;
+        const region = query.region || 'us-east-1';
+
+        if (!meetingCache[title]) {
+          meetingCache[title] = await chime
+            .createMeeting({
+              ClientRequestToken: uuidv4(),
+              MediaRegion: region
+            })
+            .promise();
+          attendeeCache[title] = {};
+        }
+        const joinInfo = {
+          JoinInfo: {
+            Title: title,
+            Meeting: meetingCache[title].Meeting,
+            Attendee: (
+              await chime
+                .createAttendee({
+                  MeetingId: meetingCache[title].Meeting.MeetingId,
+                  ExternalUserId: uuidv4()
+                })
+                .promise()
+            ).Attendee
+          }
+        };
+        attendeeCache[title][joinInfo.JoinInfo.Attendee.AttendeeId] = name;
+        response.statusCode = 201;
+        response.setHeader('Content-Type', 'application/json');
+        response.write(JSON.stringify(joinInfo), 'utf8');
+        response.end();
+        log(JSON.stringify(joinInfo, null, 2));
+      } else if (
+        request.method === 'GET' &&
+        request.url.startsWith('/attendee?')
+      ) {
+        const query = url.parse(request.url, true).query;
+        const attendeeInfo = {
+          AttendeeInfo: {
+            AttendeeId: query.attendee,
+            Name: attendeeCache[query.title][query.attendee]
+          }
+        };
+        response.statusCode = 200;
+        response.setHeader('Content-Type', 'application/json');
+        response.write(JSON.stringify(attendeeInfo), 'utf8');
+        response.end();
+        log(JSON.stringify(attendeeInfo, null, 2));
+      } else if (
+        request.method === 'POST' &&
+        request.url.startsWith('/meeting?')
+      ) {
+        const query = url.parse(request.url, true).query;
+        const title = query.title;
+        if (!meetingCache[title]) {
+          meetingCache[title] = await chime
+            .createMeeting({
+              ClientRequestToken: uuidv4()
+            })
+            .promise();
+          attendeeCache[title] = {};
+        }
+        const joinInfo = {
+          JoinInfo: {
+            Title: title,
+            Meeting: meetingCache[title].Meeting
+          }
+        };
+        response.statusCode = 201;
+        response.setHeader('Content-Type', 'application/json');
+        response.write(JSON.stringify(joinInfo), 'utf8');
+        response.end();
+        log(JSON.stringify(joinInfo, null, 2));
+      } else if (request.method === 'POST' && request.url.startsWith('/end?')) {
```

Comment:
Do we use this right now? If not just remove it

Reply:
Deleted all meeting end related code.